benchmark/primitives/primitives_test.go

```diff
@@ -92,6 +93,16 @@ func BenchmarkAtomicValue(b *testing.B) {
 	}
 }
 
+func BenchmarkAtomicValueStore(b *testing.B) {
+	c := atomic.Value{}
+	v := 123
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		c.Store(v)
```

Comment:
Do we need `v` at all?

Reply:
without v and put the constant (e.g. 123) there, it is ~11ns. While with the v, it is ~47ns. I think the compiler is doing something for constant value.

Reply:
I was going to say that's surprising, but I am honestly never surprised by things like this anymore.  Comment in the code?